Formula/x/xz.rb

```diff
@@ -4,11 +4,11 @@ class Xz < Formula
   desc "General-purpose data compression with high compression ratio"
   homepage "https://xz.tukaani.org/xz-utils/"
   # The archive.org mirror below needs to be manually created at `archive.org`.
-  url "https://github.com/tukaani-project/xz/releases/download/v5.4.6/xz-5.4.6.tar.gz"
-  mirror "https://downloads.sourceforge.net/project/lzmautils/xz-5.4.6.tar.gz"
-  mirror "https://archive.org/download/xz-5.4.6/xz-5.4.6.tar.gz"
-  mirror "http://archive.org/download/xz-5.4.6/xz-5.4.6.tar.gz"
-  sha256 "aeba3e03bf8140ddedf62a0a367158340520f6b384f75ca6045ccc6c0d43fd5c"
+  url "https://github.com/tukaani-project/xz/releases/download/v5.6.0/xz-5.6.0.tar.gz"
+  mirror "https://downloads.sourceforge.net/project/lzmautils/xz-5.6.0.tar.gz"
+  mirror "https://archive.org/download/xz-5.6.0/xz-5.6.0.tar.gz"
+  mirror "http://archive.org/download/xz-5.6.0/xz-5.6.0.tar.gz"
+  sha256 "0f5c81f14171b74fcc9777d302304d964e63ffc2d7b634ef023a7249d9b5d875"
   license all_of: [
     :public_domain,
```

Comment:
```suggestion
    "0BSD",
```

changed as of this release

Reply:
Thank you @Bo98.